### DISCUSSION TITLE
delivere

### DISCUSSION THREAD

**Discussion by JakubVanicek** [woocommerce/woocommerce#54021] [Checkout Flow]
Hello, the Czech woocommerce template does not allow you to change the Deliver section in the cart (in the order summary) to change the English word Deliver to the Czech word Delivery instead

**Comment by nielslange** [comment]
Hello @JakubVanicek 👋

Thanks for reaching out. I just switched the language to Czech and could see the issue:

<img width="477" alt="Screenshot 2025-01-06 at 18 01 11" src="https://github.com/user-attachments/assets/e2afe327-6830-436d-9089-99563de62ab6" />

The English string appears because there's no Czech translation available yet. You can provide the missing translation for this string here: https://translate.wordpress.org/projects/wp-plugins/woocommerce/dev/cs/default/?filters%5Bterm%5D=Delivery&filters%5Bcase_sensitive%5D=yes&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=untranslated&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc

Additionally, there are other untranslated strings. You can view the full list here: https://translate.wordpress.org/projects/wp-plugins/woocommerce/dev/cs/default/?filters%5Bstatus%5D=untranslated

I'm closing this discussion now, as there are no actionable items for us. But feel free to reply to this discussion, if you have more questions, or create another discussion if you have other questions or suggestions.

**Reply by JakubVanicek** [reply]
Hello Niels,
Thank you for your reply. I have added the translations, if I find something else that would be displayed in English to a potential customer, I will find the phrase in the lists and add it as well.
Free shipping means in Czech "Doručení zdarma"
Have a nice day
Jakub

**Reply by nielslange** [reply]
Hello @JakubVanicek, 

Great to hear that you've added the Czech translation. Thank you very much for this contribution. I wish you a great day as well, and as mentioned before, if you have any further questions or suggestions, just reach out. 🙌
